Sources/Sentry/SentryScope.m

```diff
@@ -265,12 +246,6 @@ - (void)applyToSession:(SentrySession *)session {
             session.user = self.userObject.copy;
         }
 
-        NSString *releaseName = [self releaseString];
-        if (nil != releaseName) {
-            // TODO: Make sure release set on options is applied to the scope so it's available now
-            session.releaseName = releaseName;
-        }
```

Comment:
`high:` should not it be like `session.releaseName = options.releaseName;` instead? I don't see how the session would take the release's name from options.

Reply:
It wasn't there yet. Now it's part of `startSession` using the options of the bound client

Reply:
got it, LGTM